README.md

```diff
@@ -1,102 +1,93 @@
 
-Ansible role for managing Advanced Intrusion Detection Environment (AIDE).
+This is an ansible role that installs and configures the [Advanced Intrusion Detection Environment (AIDE)](https://aide.github.io). For Day 2 tasks it can run integrity checks and update the AIDE database.
 
-## Requirements
+_Notice:_ This is a very early stage of a work in progress. Please use with
+extreme caution as it might break your system.
 
-Any prerequisites that may not be covered by Ansible itself or the role should
-be mentioned here.  This includes platform dependencies not managed by the
-role, hardware requirements, external collections, etc.  There should be a
-distinction between *control node* requirements (like collections) and
-*managed node* requirements (like special hardware, platform provisioning).
+## What does this role do for you?
 
-### Collection requirements
+* It ensures that the `aide` package is installed on the remote nodes
+* As an optional task it can generate the `/etc/aide.conf` file and template it out to the remote nodes
+* It initializes the AIDE database
+* The AIDE databases from the remote nodes are stored in a central directory on the controller node
+* It runs AIDE integrity checks on the remote nodes
+* It updates the AIDE databases and stores them on the controller node
 
-For instance, if the role depends on some collections and has a
-`meta/collection-requirements.yml` file for installing those dependencies, and
-in order to manage `rpm-ostree` systems, it should be mentioned here that the
- user should run
+## How does the role do that?
 
-```bash
-ansible-galaxy collection install -vv -r meta/collection-requirements.yml
-```
+* The role is controlled by using [Ansible Tags](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tags.html)
+* If you run the playbook without specifying any tag the role will change nothing on your remote nodes
+* To execute some supported use cases you need to explicitly specify one or more of the following tags
 
-on the *control node* before using the role.
+### Available tags to control and use the role
```

Comment:
Using tags is not consistent with other roles. I'd like to change this and instead use ansible variables. Can be a set of boolean vars like `aide_install`, `aide_generate_config`, `aide_init`, `aide_check`, `aide_update`. 
And we should decide what should be the minimum role's behavior. I'd say that the current install phase can be always run, and `aide_install` is then not required at all. There might be similar phases that we can drop.